samples/snippets/requirements-test.txt

```diff
@@ -0,0 +1,2 @@
+google-cloud-compute
```

Comment:
No need to add this here since the tests will install both `requirements.txt` and `requirements-test.txt`.